src/mmw/js/src/modeling/tr55/quality/views.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
+
+var Marionette = require('../../../../shim/backbone.marionette'),
+    coreModels = require('../../../core/models.js'),
```

Comment:
Just as a general comment we can omit the attributes argument if it's empty so `new coreModels.DataCollection()` should by valid.